web/src/sidebar_ui.ts

```diff
@@ -128,6 +128,10 @@ export function initialize(): void {
         e.preventDefault();
         e.stopPropagation();
 
+            // Toggle the visibility of the right side icon
+        $("#rightside-open-dot").toggle(); 
+        $("#rightside-close-dot").toggle();
```

Comment:
We should clearly hide one and show the other here and not just use `toggle` for code reading clarity.